File: ncDados/dVendasPorVendedor.vb

Namespace nsVenda

    Public Class ColecaoVendasPorVendedor
        Inherits List(Of dVendasPorVendedor)
    End Class

    Public Class dVendasPorVendedor
        Public Property _nome As String
        Public Property _totalVendas As Integer
        Public Property _quantidadeProdutos As Integer
        Public Property _valorTotalVendas As Decimal
        Public Property _ticketMedio As Decimal
        Public Property _percentualAtingimento As Decimal
        Public Property _data As DateTime
        Private Property _dataFim As DateTime

        Public Property Nome() As String
            Get
                Return _nome
            End Get
            Set(ByVal value As String)
                _nome = value
            End Set
        End Property

        Public Property TotalVendas() As Integer
            Get
                Return _totalVendas
            End Get
            Set(ByVal value As Integer)
                _totalVendas = value
            End Set
        End Property

        Public Property QuantidadeProdutos() As Integer
            Get
                Return _quantidadeProdutos
            End Get
            Set(ByVal value As Integer)
                _quantidadeProdutos = value
            End Set
        End Property

        Public Property ValorTotalVendas() As Decimal
            Get
                Return _valorTotalVendas
            End Get
            Set(ByVal value As Decimal)
                _valorTotalVendas = value
            End Set
        End Property

        Public Property TicketMedio() As Decimal
            Get
                Return _ticketMedio
            End Get
            Set(ByVal value As Decimal)
                _ticketMedio = value
            End Set
        End Property

        Public Property PercentualAtingimento() As Decimal
            Get
                Return _percentualAtingimento
            End Get
            Set(ByVal value As Decimal)
                _percentualAtingimento = value
            End Set
        End Property


        Public Property Data() As DateTime
            Get
                Return _data
            End Get
            Set(ByVal value As DateTime)
                _data = value
            End Set
        End Property

        Public Property DataFim() As DateTime
            Get
                Return _dataFim
            End Get
            Set(ByVal value As DateTime)
                _dataFim = value
            End Set
        End Property
    End Class

End Namespace

File: ncModelos/dVendasPorVendedor.cs
using System;
using System.Collections.Generic;

namespace ncModelos
{
    public class ColecaoVendasPorVendedor : List<dVendasPorVendedor>
    {
    }

    public class dVendasPorVendedor
    {
        private string _nome;
        private int _totalVendas;
        private int _quantidadeProdutos;
        private decimal _valorTotalVendas;
        private decimal _ticketMedio;
        private decimal _percentualAtingimento;
        private DateTime _data;
        private DateTime _dataFim;

        public string Nome
        {
            get => _nome;
            set => _nome = value;
        }

        public int TotalVendas
        {
            get => _totalVendas;
            set => _totalVendas = value;
        }

        public int QuantidadeProdutos
        {
            get => _quantidadeProdutos;
            set => _quantidadeProdutos = value;
        }

        public decimal ValorTotalVendas
        {
            get => _valorTotalVendas;
            set => _valorTotalVendas = value;
        }

        public decimal TicketMedio
        {
            get => _ticketMedio;
            set => _ticketMedio = value;
        }

        public decimal PercentualAtingimento
        {
            get => _percentualAtingimento;
            set => _percentualAtingimento = value;
        }

        public DateTime Data
        {
            get => _data;
            set => _data = value;
        }

        public DateTime DataFim
        {
            get => _dataFim;
            set => _dataFim = value;
        }
    }

}
